frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/_configure.scss

```diff
@@ -14,3 +14,7 @@
 .ceph__multus-dropdown-button {
   min-width: 18em;
 }
+
+.ceph__multus-tech-preview-badge--margin {
+  margin: calc(var(--pf-global--spacer--xs) * -1) 0 0 calc(var(--pf-global--spacer--sm) * -2);
```

Comment:
Why negative margins?

Reply:
![Screenshot from 2021-07-14 16-14-01](https://user-images.githubusercontent.com/39404641/125732122-0facc824-d3ae-4dfc-bc9b-e04b6c8aa430.png)

This is how badge was looking earlier, applied negative margin to top and left so that badge is aligned with the multus radio button.

Reply:
ack.